### PR TITLE
broker: swap default broker from eats to http

### PR DESCRIPTION
**Pull request by vtolstov**
Signed-off-by: Vasiliy Tolstov <v.tolstov@unistack.org>

### CODE REVIEW

**Comment by vtolstov** [comment]
@asim why we have Handle option in go-plugins/broker/http ? i don't see any usage of it

**Comment by asim** [comment]
It was to enable adding a handler for things like prometheus

File: config/cmd/cmd.go [hunk 1]
```diff
@@ -318,7 +318,7 @@ var (
 	DefaultBrokers = map[string]func(...broker.Option) broker.Broker{
 		"service": brokerSrv.NewBroker,
 		"memory":  memory.NewBroker,
-		"nats":    nats.NewBroker,
```
Comment (asim) [inline config/cmd/cmd.go:321]: Still need this

**Comment by vtolstov** [comment]
do we need this option too? i'm ask because this is not default broker option, but it exposed in broker package

**Comment by vtolstov** [comment]
may be move it to go-micro/broker/http ?

File: config/cmd/cmd.go (hunk 1 above)
Comment (asim) [reply config/cmd/cmd.go:321]: Actually never mind, we'll move the imports into micro itself

**Comment by asim** [comment]
yea add the options

**Comment by vtolstov** [comment]
done

File: broker/http.go [hunk 2]
```diff
@@ -0,0 +1,709 @@
...
+}
+
+type httpEvent struct {
+	m   *Message
+	t   string
+	err error
+}
+
+var (
+	DefaultSubPath   = "/_sub"
```
Comment (asim) [inline broker/http.go]: /

File: broker/http.go [hunk 3]
```diff
@@ -0,0 +1,709 @@
...
+	serviceName      = "go.micro.http.broker"
```
Comment (asim) [inline broker/http.go]: s/go//

File: broker/http.go [hunk 4]
```diff
@@ -0,0 +1,709 @@
...
+
+	var m *Message
+	if err = h.opts.Codec.Unmarshal(b, &m); err != nil {
+		errr := merr.InternalServerError("go.micro.broker", "Error parsing request body: %v", err)
+		w.WriteHeader(500)
+		w.Write([]byte(errr.Error()))
+		return
+	}
+
+	topic := m.Header[":topic"]
```
Comment (asim) [inline broker/http.go]: s/:topic/Micro-Topic/

**Comment by asim** [comment]
Merge?

**Comment by vtolstov** [comment]
waiting for ci

**Pull request merged**